Week1/solution/solution5.py

```diff
@@ -0,0 +1,7 @@
+#Write a python script to enter any string and count vowel.
+n=input("enter string :")
+vowels=0
+for i in range(len(n.lower())):
+    if n[i] in ['a','e','i','o','u']:

```

Comment:
Here , if I input any string which has capital letters in it then they are not counted as vowel... So can you resolve this error..?

Reply:
That why I used lower() in loop so that lower() convert the inputted string into lower case